resources/file_list_form.ui

```diff
@@ -30,60 +30,180 @@
     <number>2</number>
    </property>
    <item>
-    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,1,0">
+    <layout class="QHBoxLayout" name="horizontalLayout_8">
+    <!-- <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,1,0">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
-     </property>
+     </property> -->
```

Comment:
Do we want to keep this comment for backup?
Was this change made by QTDesigner or manually?

Reply:
Good catch, I think this was a leftover merge conflict I was deciding between. It has been removed.